### DISCUSSION TITLE
系统设置为使用HDR后，播放暂停时画面颜色变化

### DISCUSSION THREAD

**Discussion by sunbeams001** [hooke007/mpv_PlayKit#116] [mpv 通用]
系统 windows 10 21H2
显示器支持HDR400
懒人包 20220403

开启windows的HDR设置后出现此问题，具体表现为播放暂停时画面颜色变化，饱和度升高，继续播放后恢复正常

**Comment by hooke007** [comment]
我已经很久没用过hdr了，也不清楚你是gpu还是gpu-next，gpu的bug估计也不会有人再修了，如果是gpu-next的问题，你需要向官方反馈

**Reply by sunbeams001** [reply]
是默认的gpu，改成next之后直接全程变色了

**Reply by hkiceflow** [reply]
这问题我也有，开去色带也会有

**Comment by hkiceflow** [comment]
一样的，开hdr就有着问题，用去色带也会有

**Comment by ccboxes** [comment]
这个问题会在Windows HDR+显示器HDR+播放SDR内容出现，似乎在暂停时Windows的SDR-->HDR失效了，解决方法是设置
`target-prim = bt.2020`
`target-trc = pq`
在mpv中把SDR内容色域和gamma预先转换为HDR10标准
@hkiceflow @hooke007

**Reply by hooke007** [reply]
楼上说的不是sdr片源
